### DISCUSSION TITLE
Allow sending message from the JS website

### DISCUSSION THREAD

**Discussion by sjcotto** [chatwoot/chatwoot#8998] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

Not sure if there is any workarround, but we need to show a greeting message before the user sends the first message, saww the greetings works differently, also saw that the hook for widget triggered is sent before the user clicks on "Start conversation", the actual "Start conversation" button is not sending any requests. 

One option is that if we could send a message from the website into the widget, something like

$chatwoot.sendMessage("...") 

is this possible?

### Describe the solution you'd like

Have a way to send messages or display greetings after start conversation button is clicked

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-849/allow-sending-message-from-the-js-website">PR-849 Allow sending message from the JS website</a></p>

**Comment by pranavrajs** [comment]
Have you tried campaigns? https://www.chatwoot.com/docs/product/features/campaigns/ongoing
